region/brinstar/green/Early Supers Room.json

```diff
@@ -253,8 +175,12 @@
     {
       "from": 4,
       "to": [
-        {"id": 1},
-        {"id": 2}
+        {
+          "id": 1,
+          "devNote": "FIXME: A leaveCharged might be possible here from above? If so, it would be very tight."
```

Comment:
I'd need another 2 or 3 frames to leave shinecharged, with none remaining.  But I can leave in spark.
It technically possible to start running from within the leftmost shutter and leave shinecharged. Not reliable and takes exact tap timings.

Reply:
I'm curious where/how you do the spark. Do you land on the floor and then jump up, or on the platform and then jump down? Or spin jump off the crumble and spark mid-air from there? How many shinespark frames?

Reply:
Thanks for the video, I added the strat now. I assume it should be notable because it seems super precise. I put a "canQuickCrumbleEscape" requirement on it, been thinking that we should rename this to "canQuickCrumble" or "canCrumbleQuickDrop" and define it to just be the quick drop; "crumble escape" seems overly niche to be a tech.